### ISSUE TITLE
HSB to white

### ISSUE THREAD

**Issue by Legzeg** [atuline/FastLED-Demos#16]
Thanks for this all-in-one FastLED collection (I mean "seirlight" pack). It's realy cool.
I've changed the code to use it in openhab through MQTT protocol. All the modes works great!!
Solid color picker works too, but I have a problem whith solid white color.
"Mode 1" sets it perfect, but when I use HSB Values it never go to white and stays in previously selected hue even when brightness goes to 100 and saturation goes to 0. 
In other words I can't set the lightly green color (for example), just standart green wich can be dimmed only by brighness trigger.

**Comment by atuline** [comment]
Unfortunately, FastLED does have some known problems when values are dim. Drives me crazy.
